electrum/lnpeer.py

```diff
@@ -1885,17 +1924,98 @@ def verify_signature(tx, sig):
                 to_remote_amount = closing_tx.outputs()[to_remote_idx].value
                 transaction.check_scriptpubkey_template_and_dust(their_scriptpubkey, to_remote_amount)
 
-            # Agree if difference is lower or equal to one (see below)
-            if abs(our_fee - their_fee) < 2:
+            # 1. check fees
+            # if fee_satoshis is equal to its previously sent fee_satoshis:
+            if our_fee == their_fee:
+                # SHOULD sign and broadcast the final closing transaction.
+                break  # we publish
+
+            # 2. at start, adapt our fee range if we are not the channel initiator
+            fee_range_received = cs_payload['closing_signed_tlvs'].get('fee_range')
+            self.logger.info(f"Received fee range: {fee_range_received} and fee: {their_fee}")
+            # The sending node: if it is not the funder:
+            if fee_range_received and not is_initiator and not fee_range_sent:
+                # SHOULD set max_fee_satoshis to at least the max_fee_satoshis received
+                fee_range['max_fee_satoshis'] = max(fee_range_received['max_fee_satoshis'], fee_range['max_fee_satoshis'])
+                # SHOULD set min_fee_satoshis to a fairly low value
+                # TODO: what's fairly low value? allows the initiator to go to low values
+                fee_range['min_fee_satoshis'] = fee_range['max_fee_satoshis'] // 2
+
+            # 3. if fee_satoshis matches its previously sent fee_range:
+            if fee_range_sent and (fee_range_sent['min_fee_satoshis'] <= their_fee <= fee_range_sent['max_fee_satoshis']):
+                # SHOULD reply with a closing_signed with the same fee_satoshis value if it is different from its previously sent fee_satoshis
+                if our_fee != their_fee:
+                    our_fee = their_fee
+                    send_closing_signed()  # peer publishes
+                    break
+                # SHOULD use `fee_satoshis` to sign and broadcast the final closing transaction
+                else:
+                    our_fee = their_fee
```

Comment:
this assignment is not needed..

Reply:
that's true, I was translating the spec proverbially (`SHOULD use fee_satoshis`)